src/view/block_library_view.js

```diff
@@ -18,5 +18,28 @@
  * limitations under the License.
  */
 
-class WorkspaceConfigController {
-}
\ No newline at end of file
+/**
+ * @fileoverview BlockLibraryController controls user interaction with the
+ * block library, which is where developers can define and edit new blocks.
+ * New blocks are updated into the NavTreeView upon creation.
```

Comment:
I'm not clear what the library view is, if it isn't part of the nav tree.  Fine for now.